src/Psl/Internal/Loader.php

```diff
@@ -232,7 +232,9 @@ final class Loader
         'Psl\Str\concat',
         'Psl\Str\contains',
         'Psl\Str\contains_ci',
-        'Psl\Str\encoding',
+		'Psl\Str\detect_encoding',
+		'Psl\Str\convert_encoding',
+		'Psl\Str\is_utf8',
```

Comment:
please use spaces instead of tabs ( as per our coding standards )

you can use `composer cs:fix` before commiting to fix the coding standards automatically :)